js/app/historyStore.js

```diff
@@ -49,6 +49,13 @@ const HistoryStore = new Lang.Class({
         'current-set': GObject.ParamSpec.object('current-set', 'current-set', 'current-set',
             GObject.ParamFlags.READABLE, SetObjectModel.SetObjectModel),
         /**
+         * Property: current-subset
+         *
+         * The model for the current subset in the history store.
+         */
+        'current-subset': GObject.ParamSpec.object('current-subset', 'current-subset', 'current-subset',
```

Comment:
Is it a problem that this will always be `null` when not using the CourseHistoryStore? Maybe move the logic that sets it from the CourseHistoryStore up into here?

(Or, now that I think about it some more, maybe it would be better to add a new `current-toplevel-set` property, with a corresponding property to track it in the selection, since then `current-set` would have the same semantics in both CourseHistoryStore and the other HistoryStores?)